### ISSUE TITLE
Version 3.3.0 doesn't create SchemaVersions table

### ISSUE THREAD

**Issue by cdimitrov** [DbUp/DbUp#142]
Upon PerformUpgrade() I get the following message: Invalid object name 'SchemaVersions'.

The issue is not present in version 3.2.1. 
A change wasn't announced in the release notes, so I consider this as a bug.

**Comment by overforkover77** [comment]
I had the same issue and also thought this was a bug.  

In my case, I had discovered that my script file (only a single file as this is a test) was not marked as an Embedded Resource.

Once I had set this, the SchemaVersions table was properly created.

**Comment by JakeGinnivan** [comment]
I forgot to reply to this, I did look into it. The tests are showing the table created, can you provide a repro?

It may be a bug when there are no scripts because we removed a duplicate check of the table on startup and it may have introduced the issue.

**Comment by coridrew** [comment]
I am having the same issue. I actually can't "un-reproduce" it. When I follow the default instructions on a new db, or on an existing one, the SchemaVersions table does not get created. I, too, am using 3.3.0. 

I wil try now, downgrading to 3.2.1.

**Comment by helephant** [comment]
I've had the same issue with 3.3.0. 

I got the error when for some reason the database could not be accessed, for example because the database name was wrong.

**Comment by dave600b** [comment]
We discovered that we had this issue when running in debug on some developers machines. It was due to having the "Debug -> Exceptions -> Common Language Runtime Exceptions -> Throw" setting ticked.

**Comment by SergeyKulgan** [comment]
Had the same issue, have 1 valid update script. Version 3.3.0 doesn't create SchemaVersions table. Downgraded to 3.2.1, all works.

**Comment by henkmeulekamp** [comment]
Same issue here. Just downgraded from 3.3.4 back to 3.2.1 and that fixed the issue.

Running this in debug mode in visual studio.

**Comment by JakeGinnivan** [comment]
@henkmeulekamp what type of database?

**Comment by henkmeulekamp** [comment]
@JakeGinnivan Tried several things after going back to 3.3.4; removing tables. removing schema name, debug, release build. Cannot reproduce the same error message. Yesterday I found this exception, which google lead me to the 3.2.1 downgrade, which worked right away. Now cant reproduce it.. 

I'm on SQL2014,  `User ID=**;Password=**;Initial Catalog=**;data source=**;`

Maybe I did something else wrong. Well, all happy now :+1: . Working on the latest version ;-)
